.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Backpack Docs CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [rebuild]
```

Comment:
This is what we'll use to trigger this workflow remotely via a curl from other workflows

Reply:
Oh nice!